apps/meeting/src/containers/MeetingEventObserver/index.tsx

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we need to remove the year in this copyright? It's a new created file.

Reply:
Fixed it thanks for the catch!